docs/docs/installation/environment-set-up.mdx

```diff
@@ -138,12 +138,7 @@ to the following restrictions:
   to use the [duckling entity extractor](../components.mdx#ducklingentityextractor) we recommend a cloud deployment of
   duckling. Progress on this can be tracked on the
   [Duckling project](https://github.com/facebook/duckling/issues/695).
-- You cannot run Rasa in a Docker container on Apple Silicon, only native installations
```

Comment:
Two weeks ago I could not run Rasa with python 3.7 because `scipy` package v1.7.3 cannot be compiled on Mac M1 chip on python 3.7. I also could not run Rasa on `Ubuntu 22.04` for `arm64` on python 3.7, issues with tensorflow.
If this is still a case we should document it.

Reply:
Regarding `scipy`, I suggest fixing it in `pyproject.yml`.
Regarding Rasa on `Ubuntu 22.04` for `arm64` - can you tell me more about the issue with TensorFlow? (`tensorflow==2.11.1` doesn't support `Python 3.7`, we have a fix for this issue)

Reply:
`scipy` now has version condition `<1.7.3` for `Python 3.7`